svgcard.js

```diff
@@ -119,7 +121,9 @@ function renderItem(item, parent) {
       i.size(i.width() * ratio, i.height() * ratio)
        .center(width/2, ((height-35)/2)+35)
        .dmove(item.nudgeX, item.nudgeY);
-    }).css('mix-blend-mode', 'multiply');
+    }).css('mix-blend-mode', 'multiply')
+    .css('transform','rotate('+item.rotate+'deg)')
+    .css('transform-origin','center');
```

Comment:
Do this using the `.transform()` method: https://svgjs.dev/docs/3.0/manipulating/#transform-as-setter